core/src/components/backdrop/backdrop.ionic.scss

```diff
@@ -0,0 +1,7 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./backdrop";
+
+:host {
+  background-color: globals.$ionic-color-base-black;
+  backdrop-filter: blur(2px);
```

Comment:
@tanner-reits I didn't found this blur on the Figma? Was it required by UX?

Reply:
@BenOsodrac It was in the styles listed in dev mode:
```
opacity: 0.7;
background: var(--color-base-black, #111);
backdrop-filter: blur(2px);
```

Reply:
Its just that looking at the actual UI doesn't look it has blur. Also, from the snapshots, it seems its only working on Chrome.

Maybe it might need the `-webkit-` for it to work, as mentioned in this [this source](https://www.reddit.com/r/css/comments/12jhh10/solution_backdrop_filter_blur_doesnt_work_in/).

Reply:
@BenOsodrac So the issue with the snapshots is that Playwright doesn't seem to correctly handle the `backdrop-filter` property (https://github.com/microsoft/playwright/issues/28363). If you open the [preview for the sheet modal](https://ionic-framework-ogvfcq66v-ionic1.vercel.app/src/components/modal/test/sheet?ionic:theme=ionic) in Safari, Chrome, or Firefox, the blur effect is there. 

Seems like this is something we've ran into before with other components, and not really anything we can do about it. Might get resolved with new binaries a part of the iOS 18 release, but can't be certain.

Reply:
Got it, thanks, great work 🚀